repos/msbuild.proj

```diff
@@ -33,6 +33,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <RepositoryReference Include="core-setup" />
```

Comment:
Update `known-good` too? Not necessary but it'd be annoying to get thrown off by the difference.

Reply:
Fixed.